src/main/resources/add-ons-to-index.json

```diff
@@ -152,6 +158,9 @@
       "uid": "org.openmrs.module.appframework",
       "type": "OMOD",
       "name": "App Framework Module",
+       "tags": [
+        "ui-framework"
```

Comment:
This tag refers specifically to 1 module, also not helpful to tag something according to the module name.